source/style-guide/includes/terms/o.rst

```diff
@@ -49,11 +49,16 @@ O
 
    on-premises
      **off-premises**
-     Use these terms to distinguish local computing (in which
-     resources are located on a customer’s own site) from remote
+
+     :icon-fa4:`times-circle` Avoid using these terms to distinguish
+     non-Atlas deployments. To distinguish non-Atlas deployments, use
+     :term:`self-managed`.
+
+     Use *on-premises* to distinguish local computing (in which
+     resources are located on a customer's own site) from remote
      computing (in which resources are provided partially or totally
-     through the cloud). Always hyphenate. *Premises* is plural;
-     don't use *on-premise* or *off-premise*.
+     through the cloud). Always hyphenate. *Premises* is plural; don't
+     use *on-premise* or *off-premise*.
```

Comment:
[q] Not sure how prevalent it is here, but in past companies I've also seen `on-prem` used pretty frequently - is it worth calling that out here or does this cover it?

Reply:
@stephmarie17 thanks for this comment. I added a small clarification that for consistency we should use "on-premises". Let me know if you'd like to see anything different here.

Reply:
LGTM 👍